composer.json

```diff
@@ -36,16 +36,15 @@
     "scripts": {
         "symfony-scripts": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
+            "app/console cache:clear --env prod",
```

Comment:
Does this make a difference to clear the cache in both environments?

(The current solution ScriptHandler::clearCache() only calls "cache:clear" and only once.)